molecule/default/molecule.yml

```diff
@@ -158,14 +213,22 @@ platforms:
           - zookeeper-201
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+      - ./kafka_server_jaas.conf:/opt/kafka/jaas/kafka_server_jaas.conf
   - name: kafka2-201
     image: wurstmeister/kafka:2.11-2.0.1
     command: "start-kafka.sh"
     env:
       HOSTNAME_COMMAND: "hostname -i | cut -d' ' -f1"
       KAFKA_DELETE_TOPIC_ENABLE: "true"
-      KAFKA_ADVERTISED_PORT: "9092"
+      KAFKA_ADVERTISED_LISTENERS: SASL_PLAINTEXT://_{HOSTNAME_COMMAND}:9092
```

Comment:
Maybe we should keep the current `PLAINTEXT` listener on port 9092 and add a new `SASL_PLAINTEXT` listener on another port (9094 for instance)? It seems like we would then be able to keep doing the current test without SASL on port 9092 and use the 9094 for your new tests that uses SASL